<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_ModeFonctionnementCPU</class>
 <widget class="QMainWindow" name="ihm_ModeFonctionnementCPU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ModeFonctionnement</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="ModeFonctionnement">
      <item>
       <property name="text">
        <string>Mode Autonome</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mode Pilot√© Labotbox</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mode Terminal</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pb_send">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pb_reset_cpu">
      <property name="text">
       <string>Reset CPU</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>27</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>356</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
